spec/attributes.md

```diff
@@ -2,496 +2,509 @@
+    
+    If exactly one of the two steps above results in a type derived from `System.Attribute`, then that type is the result of the *attribute-name*. Otherwise a compile-time error occurs.
 
-If exactly one of the two steps above results in a type derived from System.Attribute, then that type is the result of the *attribute-name*. Otherwise a compile-time error occurs.
-
-\[*Example*: If an attribute class is found both with and without this suffix, an ambiguity is present, and a compile-time error results. If the *attribute-name* is spelled such that its right-most *identifier* is a verbatim identifier (§7.4.3), then only an attribute without a suffix is matched, thus enabling such an ambiguity to be resolved. The example
+[*Example*: If an attribute class is found both with and without this suffix, an ambiguity is present, and a compile-time error results. If the *attribute-name* is spelled such that its right-most *identifier* is a verbatim identifier (§7.4.3), then only an attribute without a suffix is matched, thus enabling such an ambiguity to be resolved. The example
 
+```csharp
 using System;
 
-\[AttributeUsage(AttributeTargets.All)\]\
-public class Example: Attribute\
+[AttributeUsage(AttributeTargets.All)]
+public class Example: Attribute
 {}
 
-\[AttributeUsage(AttributeTargets.All)\]\
-public class ExampleAttribute: Attribute\
+[AttributeUsage(AttributeTargets.All)]
+public class ExampleAttribute: Attribute
 {}
 
-\[Example\] // Error: ambiguity\
+[Example] // Error: ambiguity
 class Class1 {}
 
-\[ExampleAttribute\] // Refers to ExampleAttribute\
+[ExampleAttribute] // Refers to ExampleAttribute
 class Class2 {}
 
-\[@Example\] // Refers to Example\
+[@Example] // Refers to Example
 class Class3 {}
 
-\[@ExampleAttribute\] // Refers to ExampleAttribute\
+[@ExampleAttribute] // Refers to ExampleAttribute
 class Class4 {}
+```
 
-shows two attribute classes named Example and ExampleAttribute. The attribute \[Example\] is ambiguous, since it could refer to either Example or ExampleAttribute. Using a verbatim identifier allows the exact intent to be specified in such rare cases. The attribute \[ExampleAttribute\] is not ambiguous (although it would be if there was an attribute class named ExampleAttributeAttribute!). If the declaration for class Example is removed, then both attributes refer to the attribute class named ExampleAttribute, as follows:
+shows two attribute classes named `Example` and `ExampleAttribute`. The attribute [Example] is ambiguous, since it could refer to either `Example` or `ExampleAttribute`. Using a verbatim identifier allows the exact intent to be specified in such rare cases. The attribute [ExampleAttribute] is not ambiguous (although it would be if there was an attribute class named `ExampleAttributeAttribute`!). If the declaration for class `Example` is removed, then both attributes refer to the attribute class named `ExampleAttribute`, as follows:
```

Comment:
Missing code-fence:

```suggestion
shows two attribute classes named `Example` and `ExampleAttribute`. The attribute `[Example]` is ambiguous, since it could refer to either `Example` or `ExampleAttribute`. Using a verbatim identifier allows the exact intent to be specified in such rare cases. The attribute `[ExampleAttribute]` is not ambiguous (although it would be if there was an attribute class named `ExampleAttributeAttribute`!). If the declaration for class `Example` is removed, then both attributes refer to the attribute class named `ExampleAttribute`, as follows:
```